HLTrigger/HLTfilters/plugins/L1TTkMuonFilter.cc

```diff
@@ -45,18 +46,23 @@ namespace {
 L1TTkMuonFilter::L1TTkMuonFilter(const edm::ParameterSet& iConfig)
     : HLTFilter(iConfig),
       l1TkMuonTag_(iConfig.getParameter<edm::InputTag>("inputTag")),
-      tkMuonToken_(consumes<l1t::TrackerMuonCollection>(l1TkMuonTag_)),
-      qualityCut_(iConfig.getParameter<edm::ParameterSet>("qualities")) {
+      tkMuonToken_(consumes<l1t::TrackerMuonCollection>(l1TkMuonTag_)) {
```

Comment:
```suggestion
      tkMuonToken_(consumes(l1TkMuonTag_)) {
```
Simpler (..since the PR needs to be updated anyways).

Reply:
Thanks, done!